docs/_includes/generated/yarn_config_configuration.html

```diff
@@ -128,6 +122,12 @@
             <td>List&lt;String&gt;</td>
             <td>A semicolon-separated list of provided lib directories. They should be pre-uploaded and world-readable. Flink will use them to exclude the local Flink jars(e.g. flink-dist, lib/, plugins/)uploading to accelerate the job submission process. Also YARN will cache them on the nodes so that they doesn't need to be downloaded every time for each application. An example could be hdfs://$namenode_address/path/of/flink/lib</td>
         </tr>
+        <tr>
+            <td><h5>yarn.security.kerberos.additionalFileSystems</h5></td>
+            <td style="word-wrap: break-word;">(none)</td>
```

Comment:
can you put this change into a separate commit labeled "[hotfix][docs] Fix order"? Looks like someone modified the file manually once again...